system/web.php

```diff
@@ -344,16 +344,23 @@ function install() {
      */
     function start($init_database = true) {
         if ($init_database && !$this->_is_installing) {
-			$this->initDB();
-		}
+                $this->initDB();
+        }
 
-		// Set the timezone from Config
-		$timezone = Config::get('system.timezone');
-		if (empty($timezone)) {
-			$timezone = 'UTC';
-		}
-		date_default_timezone_set($timezone);
+        // Set the timezone from Config
+        $timezone = Config::get('system.timezone');
+        if (empty($timezone)) {
+                $timezone = 'UTC';
+        }
+        date_default_timezone_set($timezone);
 		
+        //check config for 'gc_maxlifetime' for the session
+        $gc_maxlifetime = Config::get('system.gc_maxlifetime')[0];
```

Comment:
You shouldn't need the [0] part, the 'system.gc_maxlifetime' should be returning an integer...